### ISSUE TITLE
SRG Request Age-1 index CVs

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [pacific-hake/hake-assessment#1067]
2023 SRG Request
8. Given the importance of the age-1 index in estimating the size of the age-classes entering the fishery, the SRG recommends that the JTC __implement updated age-1 index CVs__, when they are provided by the Survey Team, in the stock assessment model.

**Comment by cgrandin** [comment]
Was this included with the estimate this year?

**Comment by aaronmberger-nwfsc** [comment]
No.  We are still using a constant CV set to 0.50.  This request is something the survey team needs to do or at least substantially help with.  Nonetheless, it will not be addressed this year.
